composer.json

```diff
@@ -32,7 +32,7 @@
     "php": "^7.3 || ^8.0",
     "ext-openssl": "*",
     "ext-mbstring": "*",
-    "jeremeamia/superclosure": "~2.2",
+    "opis/closure": "~3.6",
```

Comment:
We'll also need to update the composer.json in the [Routing directory](https://github.com/opulencephp/Opulence/blob/3710de51f0d3dbc4a2770aea40906a66348aaa7c/src/Opulence/Routing/composer.json)

Reply:
Interesting, I thought I grepped the code base afterwards :thinking: